sphinx/builders/epub3.py

```diff
@@ -215,9 +215,37 @@ def build_navigation_doc(self, outdir, outname):
 
 def setup(app):
     # type: (Sphinx) -> Dict[unicode, Any]
-    app.setup_extension('sphinx.builders.epub')
+
+    app.setup_extension('sphinx.builders.epub2')
+
     app.add_builder(Epub3Builder)
 
+    # config values
+    '''
+    app.add_config_value('epub_basename', lambda self: make_filename(self.project), None)
+    app.add_config_value('epub_theme', 'epub', 'html')
+    app.add_config_value('epub_theme_options', {}, 'html')
+    app.add_config_value('epub_title', lambda self: self.html_title, 'html')
+    app.add_config_value('epub_author', 'unknown', 'html')
+    app.add_config_value('epub_language', lambda self: self.language or 'en', 'html')
+    app.add_config_value('epub_publisher', 'unknown', 'html')
+    app.add_config_value('epub_copyright', lambda self: self.copyright, 'html')
+    app.add_config_value('epub_identifier', 'unknown', 'html')
+    app.add_config_value('epub_scheme', 'unknown', 'html')
+    app.add_config_value('epub_uid', 'unknown', 'env')
+    app.add_config_value('epub_cover', (), 'env')
+    app.add_config_value('epub_guide', (), 'env')
+    app.add_config_value('epub_pre_files', [], 'env')
+    app.add_config_value('epub_post_files', [], 'env')
+    app.add_config_value('epub_exclude_files', [], 'env')
+    app.add_config_value('epub_tocdepth', 3, 'env')
+    app.add_config_value('epub_tocdup', True, 'env')
+    app.add_config_value('epub_tocscope', 'default', 'env')
+    app.add_config_value('epub_fix_images', False, 'env')
+    app.add_config_value('epub_max_image_width', 0, 'env')
+    app.add_config_value('epub_show_urls', 'inline', 'html')
+    app.add_config_value('epub_use_index', lambda self: self.html_use_index, 'html')
+    '''
```

Comment:
Hmm... why quoted?

BTW, it would be nice if we move common setting up scripts of EPUB builders to `_epub_base.setup()`.
For example, `sphinx.ext.mathbase` provides `setup_math()` function to derived extensions.
https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/mathbase.py#L320